syntax = "proto3";

service GoToMissionWaitingArea{
    rpc Take_Off_Connection_Failed(Empty) returns (ACK);
    rpc Take_Off_Connected_But_Failed(Empty) returns (ACK);
    rpc Take_Off_Done(Empty) returns (ACK);

    rpc Go_Up_Failed(Empty) returns (ACK);
    rpc Go_Up_Set_Settings_Failed(Empty) returns (ACK);
    rpc Go_Up_Done(Empty) returns (ACK);
}

message Empty { }
message ACK { }
